src/constants.js

```diff
@@ -34,3 +35,11 @@ export const FILE_STATUSES = {
   DISCARDED: 'DISCARDED',
   CANCELLED: 'CANCELLED',
 };
+
+export const PUBLISH_COORDINATOR_STATUSES = {
+  ERROR: 'IN_PROGRESS',
```

Comment:
@usavkov-epam is it meant to be `ERROR` or `IN_PROGRESS`,

Reply:
Thanks, @alisher-epam , good catch! Updated.